client/src/components/CreateSymptoms.component.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+export default class createSympotom extends Component {
```

Comment:
is this component for the patient profile?  reading it, it seems to set the state, then retrieve data from 'http://localhost:5000/patientEntry/' then display that data.

Reply:
Yes, it will retrieve the deeply sitting symptoms